plugins/authenticator/README.md

```diff
@@ -103,6 +103,22 @@ if (counter && counter > 0) {
 
 PRs accepted. Please make sure to read the Contributing Guide before making a pull request.
 
+## Partners
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="middle">
+        <a href="https://crabnebula.dev" target="_blank">
+          <img src="/.github/sponsors/crabnebula.svg" alt="CrabNebula" width="283">
```

Comment:
how will this be reflected in the mirror? does it copy the `.github` directory from here to the mirror?

Reply:
ffs good call, forgot about that

Reply:
ah damn, this also means the tauri/wry/tao readmes will be broken on crates.io right?

Reply:
it should work in tao and wry, the `.github` directory is included, we have a splash file

Reply:
phew 😌

Reply:
okay the published v2 plugins already use a https link so i'll do the same. then it works for the mirrors and the package regs.